sequencing/vcf_pca/dxapp.json

```diff
@@ -161,6 +162,7 @@
       "name": "twstats_out",
       "label": "TW Statistics",
       "class": "file",
+      "help": "Tracy-Widom statistics to evaluate the statistical significance of each principal component identified by pca",
```

Comment:
A file containing Tracy-Widom statistics evaluating the statistical significance of each principal component identified by PCA